content/slides/geospatial-concepts-and-terms.qmd

```diff
@@ -210,7 +224,7 @@ _TODO: how do size images so that they take up the remaining space?_
 Website demonstrates many different projections with an interactive control to change the perspective.
 :::
 
-## Standard representations of geospatial metadata
+## Standard representations of CRS information
```

Comment:
These slides show representations of CRS information, not other (potentially geospatial) metadata. E.g., rasters need a geotransform/affine matrix to position them in space.